package.json

```diff
@@ -15,6 +15,7 @@
     "@cosmjs/proto-signing": "^0.28.0",
     "@cosmjs/stargate": "^0.28.0",
     "@keplr-wallet/types": "^0.9.0-alpha.4",
+    "@ledgerhq/hw-transport-webusb": "^6.24.1",
```

Comment:
Also we need @cosmjs/ledger-amino, which we use directly (fixed in https://github.com/cosmos/cosmos-multisig-ui/pull/58)